packages/@lwc/renderer-abstract/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "@lwc/renderer-abstract",
+    "version": "2.6.1",
+    "description": "Abstract implementation of an LWC renderer",
+    "homepage": "https://lwc.dev/",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/salesforce/lwc.git",
+        "directory": "packages/@lwc/renderer-abstract"
+    },
+    "bugs": {
+        "url": "https://github.com/salesforce/lwc/issues"
+    },
+    "main": "index.js",
+    "typings": "index.d.ts",
+    "license": "MIT",
+    "scripts": {},
+    "files": [
+        "index.*"
+    ],
+    "publishConfig": {
+        "access": "public"
+    }
+}
```

Comment:
Technically we don't _need_ to publish `@lwc/renderer-abstract`, since we only need it at build time. But I guess we might as well? It's just an `index.js` file and a `index.d.ts` file.